Sources/ResilientDecoding/ResilientRawRepresentable.swift

```diff
@@ -126,6 +126,33 @@ extension KeyedDecodingContainer {
     resilientlyDecode(valueForKey: key, fallback: nil) { $0.resilientlyDecodeArray(decodeElement: resilientlyDecodeRawRepresentable).map { $0 } }
   }
 
+  /**
+   When decoding a dictionary of `ResilientRawRepresentable` values,  elements are omitted as errors are encountered. The `decodingFallback` is never used.
```

Comment:
Why can't we use the raw representable fallback that we have?

Reply:
Well, we have to pick one behavior: we are either consistent with resilient collections (array has the same behavior) or raw representable. I think opting to omit is the more common case, and you can always create bespoke behavior by defining a custom type for the elements.